doc/lua_api.md

```diff
@@ -7493,6 +7496,8 @@ An `InvRef` is a reference to an inventory.
 * `is_empty(listname)`: return `true` if list is empty
 * `get_size(listname)`: get size of a list
 * `set_size(listname, size)`: set size of a list
+    * If `listname` is not known, a new list will be created
```

Comment:
```suggestion
    * If `listname` is not known and `size > 0`, a new list will be created
```

Otherwise these statements contradict each other if `listname` is not known, but `size` is `0`
(In other words, does the first or the second statement take precedence?)

Reply:
I'd say, the reader must be pretty stupid to think like that. A list cannot exist if size == 0, I think it's pretty clear that you can't create any list if size is 0

Reply:
okay i quickly asked a modder and it seems i'm being overly pedantic, sorry for the hiccup